### DISCUSSION TITLE
Convert custom xml test result file to allure report

### DISCUSSION THREAD

**Discussion by win5923** [allure-framework/allure2#2024] [General Discussion]
hello, I have an test result XML file with android test and I would like to know if there is a way to convert the XML file into an Allure report.

I would like to have each module as a separate test case in Allure, but I'm struggling to come up with a method to achieve that.

here is my xml test result:

https://gist.github.com/win5923/280999024dbe508268748258781b8d94

Using `allure serve D:\test\output` to view the Allure report shows nothing.

I don't have `pom.xml` file. I just want to convert this `test_result.xml` into an allure format.

**Comment by baev** [comment]
What is the tool that produces such a result?
